packages/components/src/Page/Page.tsx

```diff
@@ -101,7 +108,16 @@ export function Page({
       <Content>
         <Content spacing="large">
           <div className={titleBarClasses} ref={titleBarRef}>
-            <Heading level={1}>{title}</Heading>
+            <div>
+              <Heading level={1}>{title}</Heading>
+              <div className={styles.subheader}>
```

Comment:
@ellaestigoy we should have a conditional check here so that we only render this div and its' children if there is content for the subheader

Reply:
If we remove the margin-bottom per my comment above, I think this wrapper div can be removed altogether and we can conditionally render the `<Text>` element itself

Reply:
@chris-at-jobber with small screen, i think we need spacing in there especially if there's an action group